routes/v1.js

```diff
@@ -97,6 +101,14 @@ function addRoutes(app, peliasConfig) {
   const placeholderService = serviceWrapper(placeholderConfiguration);
   const isPlaceholderServiceEnabled = _.constant(placeholderConfiguration.isEnabled());
 
+  const changeLanguageConfiguration = new Language(_.defaultTo(peliasConfig.api.services.language, {}));
```

Comment:
I'm not sure what you mean.  Each has a different URL path and different ways of building URLs.